.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Bug report
+about: Create a report to help us improve
```

Comment:
Hi, great idea!
I have the feeling you took this template from a web app or website project.
Asking all these info about smartphones/desktop doesn't make much sense for a Python research library right?

Reply:
Hi there!
Thx for the notice 👍 I blindly used a template suggested by GitHub...